test/Confluent.Kafka.UnitTests/Admin/DisposeAsync.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Threading.Tasks;
+using Xunit;
+
+namespace Confluent.Kafka.UnitTests;
+
+public class DisposeAsync
+{
+    [Fact]
+    public async Task TestDisposeAsyncDoesNotThrow()
+    {
+        var adminClient = new AdminClientBuilder(new AdminClientConfig { BootstrapServers = "localhost:90922" }).Build();
```

Comment:
I’m not sure this tests anything. To properly verify that this method works, it would be good to have at least one iteration of the pooling loop.

Reply:
Yeah I'm not exactly sure how to test that since the loop doesn't do any actions when it's canceled. For example, it doesn't cancel pending tasks. I added a create/delete topic so at least it goes though the loop once.